str0m/src/media/mline.rs

```diff
@@ -319,11 +330,14 @@ impl MLine {
         now: Instant,
         exts: &Extensions,
         twcc: &mut u64,
+        pad_size: Option<DataSize>,
         buf: &mut Vec<u8>,
     ) -> Option<(RtpHeader, SeqNo)> {
         let mid = self.mid;
 
-        let next = if let Some(next) = self.poll_packet_resend(now) {
+        let next = if let Some(next) = pad_size.and_then(|p| self.create_padding_packet(p, now)) {
```

Comment:
I prefer this to be isolated out to `if let Some(next) = poll_packet_padding(now, pad_size)` to be uniform with the below.

Reply:
Sure 👍🏼